calico-cloud/get-started/system-requirements.mdx

```diff
@@ -1,62 +1,50 @@
 ---
 description: Review cluster requirements to connect to Calico Cloud.
-redirect_from:
-  - /install/system-requirements
-  - /get-started/connect/system-requirements
 ---
 
 # System requirements
 
-### Step 1: Verify cluster
+Before you connect your cluster to Calico Cloud, make sure your cluster meets the system requirements.
+Your cluster must already have a CNI installed before you can connect to Calico Cloud.
 
 <!--  This file is referenced in the manager UI and its location should not be changed with accounting for references from pendo/manager UI. A symlink so this file exists at install/system-requirements.  -->
 
-Verify that your cluster meets these requirements.
+## Kubernetes distributions and CNIs
 
-| Required                    | Supported                                                    |
-| --------------------------- | ------------------------------------------------------------ |
-| **Platforms**               | All of the Calico Open Source [Kubernetes deployments](https://docs.projectcalico.org/getting-started/kubernetes/), except: <br/><br/>- OpenShift<br />- IBM Cloud Kubernetes Service (IKS)<br/>- K3s <br />- Most clusters with Windows nodes. (We support Windows nodes on AKS.)<br />- RKE |
-| **Architecture**            | Only AMD64                                                   |
-| **Kubernetes version**      | - Minimum: v1.25<br/>- Maximum: v1.29<br /><br />If you have a later version than the maximum, contact [Support](https://support.tigera.io/). |
-| **CNIs**                    | - Calico CNI v3.20 to v3.27<br />- Amazon VPC CNI<br/>- Azure CNI<br/>- GKE CNI |
-| **Browsers for Manager UI** | {{prodname}} supports the latest version and "latest minus one" versions of the following browsers:<br/><br/>- Chrome<br/>- Safari<br/>- Firefox |
+Calico Cloud works with Kubernetes on self-provisioned infrastructure and on managed Kubernetes distributions.
+To use Calico Cloud for both networking and network policy, your cluster must have Calico Open Source installed before you connect to Calico Cloud.
+For most managed distributions, you can use the provider's CNI for networking and use Calico Cloud for network policy.
 
-### Step 2: Verify cluster is not managed by a Kubernetes reconciler
+| Distribution | Supported CNIs |
+| --- | --- |
+| Kubernetes on self-provisioned infrastructure | - Calico Open Source 3.26 and 3.27 |
+| Amazon Elastic Kubernetes Service | - Calico Open Source 3.26 and 3.27<br/> - Amazon VPC CNI |
+| Azure Kubernetes Service | - Calico Open Source 3.26 and 3.27<br/> - Azure CNI |
+| Google Kubernetes Engine | - Calico Open Source 3.26 and 3.27<br/> - GKE CNI |
+| Rancher Kubernetes Engine 2 | - Calico Open Source 3.26 and 3.27 |
```

Comment:
I believe each of AKS, GKE, and RKE2 have configurations that cannot be upgraded.
I think it is incomplete to leave the caveats to the "Prepare your cluster for Calico Cloud" section.

Reply:
> configurations that cannot be upgraded

What do you mean here, exactly? That you can't upgrade CC with configurations intact?

Reply:
I mean you can't install CC depending on the options or the CNI being used in the cluster.
When I said "upgraded" I meant the cluster can't have CC installed onto it.